app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] The associations between user models are set up correctly but according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) you should have foreign keys 'author id' for each association.

Reply:
@arslanbisharat, this is set by default in the schema.rb file when the associations were generated. [Here](https://github.com/muneebulrehman/blog-with-rails/blob/bd7e21138723e58715d63097c4d262dee0536690/db/schema.rb#L56) as you can see